app/controllers/notes_controller.rb

```diff
@@ -0,0 +1,47 @@
+class NotesController < ApplicationController
+
+  def index
+    find_initiative
```

Comment:
#find_initiative is repeated a few times. Could it work in a `before_action`?

Reply:
Resolved!